src/pages/PagesRouter.tsx

```diff
@@ -46,6 +47,7 @@ export const PagesRouter: React.FC = () => {
             path={'/position/:id'}
             render={({ match }) => <SinglePositionPage id={match.params.id} />}
           />
+          <Route path={'/ido'} component={IdoPage} />
```

Comment:
it was not necessary to create page, only component and storybook for it were in scope of task